static/app/components/workflowEngine/form/control/priorityControl.spec.tsx

```diff
@@ -4,14 +4,17 @@ import Form from 'sentry/components/forms/form';
 import FormModel from 'sentry/components/forms/model';
 import PriorityControl from 'sentry/components/workflowEngine/form/control/priorityControl';
 import {PriorityLevel} from 'sentry/types/group';
+import {METRIC_DETECTOR_FORM_FIELDS} from 'sentry/views/detectors/components/forms/metricFormData';
 
 describe('PriorityControl', function () {
   it('renders children', async function () {
     const formModel = new FormModel({
       initialData: {
-        'conditionGroup.conditions.0.type': 'above',
-        'conditionGroup.conditions.0.comparison': '0',
-        'conditionGroup.conditions.0.conditionResult': PriorityLevel.LOW,
+        [METRIC_DETECTOR_FORM_FIELDS.initialPriorityLevel]: 'low',
+        [METRIC_DETECTOR_FORM_FIELDS.conditionType]: 'gt',
+        [METRIC_DETECTOR_FORM_FIELDS.conditionValue]: '0',
+        [METRIC_DETECTOR_FORM_FIELDS.mediumThreshold]: '',
+        [METRIC_DETECTOR_FORM_FIELDS.highThreshold]: '',
```

Comment:
could these live in a nested `conditions` array, then we add / remove the thresholds based on the conditions that are set? that way it's really easy to use the select on the thresholds and update.

just curious, do we have to have everything be in `.` notation to do nesting? that makes it feel fairly difficult to grok this.